client.py

```diff
@@ -0,0 +1,94 @@
+"""
+Implement a tkinter client who communicates with a fastapi server
+and executes commands
+"""
+
+import requests
```

Comment:
What is `requests`? A developer wanting to run your code should know what third-party packages he has to install. Add requirements.txt file with the requirements for the project. Read on the web about how this file should look.